contracts/database/orm/factory.go

```diff
@@ -0,0 +1,12 @@
+package orm
+
+//go:generate mockery --name=Factory
+type Factory interface {
+	Count(count int) Factory
+	Raw() any
+	Create() error
+	CreateQuietly() error
+	Make() Factory
+	Model(value any) Factory
+	NewInstance(attributes ...map[string]any) Factory
```

Comment:
S: This method needs to be a internal method.

Reply:
Okay i have made `NewInstance` as an internal method.